multi-agent/adapters/inbound/flask/endpoints/blueprints.py

```
@@ -1,393 +1,442 @@
+  BlueprintNotFoundError      → 404
+  VersionNotFoundError        → 404
+  BlueprintAccessDeniedError  → 403
+  ConcurrentModificationError → 409
+  RuntimeError (not configured) → 501
+  BlueprintError (other)      → 500
+"""
+
+from __future__ import annotations
+
+import functools
+from typing import Any, Callable, Dict, Optional, Tuple
```

Ruff (0.15.17)

[warning] Import from `collections.abc` instead: `Callable`

Import from `collections.abc`

(UP035)